kernel/entry.S

```diff
@@ -0,0 +1,7 @@
+global _start
+
+section .text
+bits 32 ; tell NASM this is 16 bit code
```

Comment:
But this is 32 bit code :wink:. Is a `bits` directive necessary here since in the Makefile we specify it should be assembled as `elf32`